packages/Python/lldbsuite/test/lang/swift/variables/uninitialized/Makefile

```diff
@@ -1,7 +1,3 @@
 LEVEL = ../../../../make
 SWIFT_SOURCES := main.swift
 include $(LEVEL)/Makefile.rules
-
-cleanup:
```

Comment:
@adrian-prantl I'm under the impression this is not needed anymore after you implemented a general `clean` rule in the top level makefile, but please correct me if I'm wrong.

Reply:
This particular `rm Makefile` is a very odd rule.
The default rule will definitely clean any `.d` files.